coffit/cli.py

```diff
@@ -15,31 +21,47 @@ def db():
 
 
 @db.command('init')
-@click.option('--url', default=config.DATABASE_URL,
-              help=('Database URL. '
-                    'eg. "postgresql+psycopg2://user:pass@localhost/coffit"'))
-def init_db(url):
+@click.pass_context
+def db_init(ctx):
+    """Create db."""
+    url = config.DATABASE_URL
     if database_exists(url):
         click.secho('Database %s already exists...' % url, fg='yellow')
-        click.confirm('Drop and create new?', default=False, abort=True)
-        click.secho('Dropping %s...' % url, fg='red')
-        drop_database(url)
+        ctx.abort()
     click.secho('Creating %s...' % url, fg='blue')
     create_database(url)
     click.secho('Creating tables...', fg='blue')
     Base.metadata.create_all(engine)
     click.secho('Database initialized!', fg='green')
 
 
+@db.command('drop')
+@click.option('--yes-i-know', is_flag=True, expose_value=False,
```

Comment:
You should consider using [`click.confirmation_option`](http://click.pocoo.org/5/api/#click.confirmation_option) instead.